### ISSUE TITLE
Zombies now missing after using custombuildings

### ISSUE THREAD

**Issue by robotstar78** [Da0ne/DZMods#108]
Hey.

So we now have custom buildings on the server - created using Offline mode and exported onto the server at the bottom of the tunables.c . The buildings have all gone in fine (a couple have shifted slightly, but nothing too major) however, zombies are now not spawning. I don't mean the zombie hordes, I mean general normal zombie spawns. Anyone had this issue?

**Comment by GravityWolfNotAmused** [comment]
No, I don't believe so. That's odd, I keep it on all the time in my testing, and there were zombies in my server, I went to NWAFN and fought some.

**Comment by robotstar78** [comment]
It’s an odd one. The zombies have actually all come back - standard dayz glitchiness I guess. However l, I can’t stop some of the buildings shifting out of place when they’re brought into our server. Is there some sort of collision thing going on? It’s happening with a lot of fences we place down. Some just don’t want to sit where I put them. All works fine in offline mode, and then when I export they’ve all moved again.

**Comment by GravityWolfNotAmused** [comment]
Let me look

**Comment by GravityWolfNotAmused** [comment]
I think it's from me setting it's direction that it's facing. Can you get that information from the editor? Because I can just expand the object more.

**Comment by GravityWolfNotAmused** [comment]
Try deleting obj.setDirection(obj.getDirection());
And let me know if that helps.

**Comment by GravityWolfNotAmused** [comment]
@robotstar78 No word?

**Comment by robotstar78** [comment]
Hey chap, sorry for the delay. I headed to bed after I messaged you. Let me have a quick look now and I'll let you know

**Comment by GravityWolfNotAmused** [comment]
if that's the case than we will release it with out that: [here](https://github.com/Da0ne/DZMods/blob/f388f344f2adf0d66a618abf1535cfbd66a98af2/mpmissions/DayZSurvival.chernarusplus/ScriptedMods/Modules/CustomBuildings/BuildingSpawner.c#L42)

That line.

**Comment by GravityWolfNotAmused** [comment]
Thank you so much! I can do some testing on this. I will get back to you as soon as I get information.

**Comment by robotstar78** [comment]
Thanks chap. I've tried that and also updated the client-side mod from the workshop. The issues still persist however. I'm wondering if it's an export problem on Community offline mode then. I've raised a ticket there too! Thanks so much for your help.

**Comment by GravityWolfNotAmused** [comment]
Welcome
